src/main/webapp/i18n/de/learningPath.json

```diff
@@ -4,12 +4,14 @@
             "navigation": {
                 "nextButton": "Weiter",
                 "previousButton": "Zurück",
+                "completeButton": "Abschließen",
                 "overview": {
                     "title": "Lernpfad Kompetenzen",
                     "showCompetenciesGraphButton": "Kompetenzgraph",
                     "emptyLearningObjectsLabel": "Diese Kompetenz enthält bisher noch keine Lernobjekte!",
                     "nextLearningObjectOnPathLabel": "Nächste"
-                }
+                },
+                "recapLabel": "Lernpfadobjekte wiederholen"
```

Comment:
Lernpfadobjekte, is there a better word? 🤔

Reply:
Done 🇹🇭